packs/equipment-effects/effect-magnetic-shot-greater.json

```diff
@@ -0,0 +1,68 @@
+{
+    "_id": "oGsTuU8u0lnpSmnE",
+    "img": "icons/commodities/tech/bearing-steel-green.webp",
+    "name": "Effect: Magnetic Shot (Greater)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.JqKNiEJxSZX1sD06]{Magnetic Shot (Greater)}</p>\n<p>The activated ammunition grants a +2 circumstance bonus to attack rolls against such targets. Due to magnetic acceleration, the ammunition deals four additional weapon damage dice and is deadly d12.</p>"
```

Comment:
These links are malformed.

Reply:
Thanks for the ammunition examples. I'll take another pass to set them up that way, and I'll take a look at the links. I was just setting up the links via drag + drop. I'll check the contributing page again for the correct practice

Reply:
Ok, the links were removed altogether and I set these up as rule elements directly on the ammunition itself. Thanks for pointing me in the right direction!